src/Logger.js

```diff
@@ -108,6 +108,10 @@ class Logger {
       this.lines++;
     }
   }
+
```

Comment:
Internal _log function to help with stubbing, and to avoid stubbing console.log directly (nasty side-effects).